### PR TITLE
feat(docs): for run services health checks

### PR DESCRIPTION
**Pull request by NesManrique**

### CODE REVIEW

File: docs/product/run.mdx [hunk 1]
```diff
@@ -26,4 +26,15 @@ Nhost Run offers several key advantages for running workloads alongside your pro
...
 
-
+## Guides
+
+<CardGroup cols={4}>
+  <Card title="Getting Started" icon="square-1" href="../guides/run/getting-started">
+  </Card>
+  <Card title="Configuration" icon="square-2" href="../guides/run/configuration">
+  </Card>
+  <Card title="Resources" icon="square-4" href="../guides/run/resources">
```
Comment (dbarrosop) [inline docs/product/run.mdx:36]: aren't this out of order?

File: docs/product/run.mdx [hunk 2]
```diff
@@ -26,4 +26,15 @@ Nhost Run offers several key advantages for running workloads alongside your pro
...
+  </Card>
+  <Card title="CLI & CI Deployments" icon="square-3" href="../guides/run/cli-deployments">
```
Comment (dbarrosop) [inline docs/product/run.mdx:38]: aren't we missing the new healthcecks page?

File: docs/guides/run/health-checks.mdx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+title: Healthchecks
+description: description
+icon: heart-pulse
+---
+
+## Configuration
+
+Below you can find an example with a toml configuration file for a service with healthcheck enabled.
+
+```toml
```
Comment (dbarrosop) [inline docs/guides/run/health-checks.mdx:11]: I'd add here only the relevant bits for the healthchecks, for a full example we already have the cli.

File: docs/product/run.mdx (hunk 2 above)
Comment (NesManrique) [reply docs/product/run.mdx:38]: I hand picked some pages but I could add all of them if that's better.

File: docs/guides/run/health-checks.mdx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+## Healthcheck behaviour
+
+The healthcheck endpoint must be available at `/healthz` on the port specified in `healthCheck.port`. If your container image exposes multiple ports only one of them can be used for healthchecks.
+
+The endpoint must return `200` status code within `5` seconds for the service to be considered healthy. If the service fails to respond correctly `3` times in a row it will be considered not healthy and will be restarted with an exponential backoff delay (10s, 20s, 40s, …), that is capped at five minutes.
+
+With `healthCheck.initialDelaySeconds` you can specify the number of seconds to wait before sending the first healthcheck request, this is useful if the service needs some time to start up. The default value is `30` seconds.
+
+With `healthCheck.probePeriodSeconds` you can specify the number of seconds to wait between healthcheck requests. The default value is `10` seconds.
```
Comment (dbarrosop) [inline docs/guides/run/health-checks.mdx:52]: is the default value 10s? It's a bit low, 60s would probably be better for a default.

File: docs/guides/run/health-checks.mdx (hunk 3 above)
Comment (NesManrique) [reply docs/guides/run/health-checks.mdx:11]: ok

File: docs/guides/run/health-checks.mdx (hunk 4 above)
Comment (NesManrique) [reply docs/guides/run/health-checks.mdx:52]: Yeah I think so too, will fix that on BE repo later then.

File: docs/product/run.mdx (hunk 2 above)
Comment (dbarrosop) [reply docs/product/run.mdx:38]: yeah, let's do all for now

File: docs/product/run.mdx (hunk 1 above)
Comment (NesManrique) [reply docs/product/run.mdx]: solved

File: docs/product/run.mdx (hunk 2 above)
Comment (NesManrique) [reply docs/product/run.mdx]: done

File: docs/guides/run/health-checks.mdx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+## Configuration
+
+Below you can find an example with a toml configuration file for a service with healthcheck enabled.
+
+```toml
+name = 'mysvc'
+.
+.
+.
+[[ports]]
```
Comment (dbarrosop) [inline docs/guides/run/health-checks.mdx:16]: I'd remove this as this is unrelated. I may also mention in the docs that the port you use for the healthchecks doesn't need to be exposed to the outside world.

**Review by dbarrosop — APPROVED** [review]

File: docs/guides/run/health-checks.mdx (hunk 5 above)
Comment (NesManrique) [reply docs/guides/run/health-checks.mdx]: fixed
